### PR TITLE
Randomizer Made

### PR DESCRIPTION
**Pull request by AndrewMacAskill**
Randomizer Make pls Review

### CODE REVIEW

File: rps.js [hunk 1]
```diff
@@ -6,8 +6,14 @@ const readline = require('readline').createInterface({
 // asks the user to enter their move and gets input
 readline.question('Enter your move (r/p/s)', move => {
     console.log(`Your move is ${move}`);
+    const computermove = getRandomInt(3)
+    const cmoves = ["r","p","s"]
+    const cmovesrps = cmoves [computermove]
+    console.log (`move is ${cmovesrps}`)
 });
-
+function getRandomInt(max) {
```
Comment (nick-simone) [inline rps.js]: use arrow function pls

File: rps.js [hunk 2]
```diff
@@ -6,8 +6,14 @@ const readline = require('readline').createInterface({
 // asks the user to enter their move and gets input
 readline.question('Enter your move (r/p/s)', move => {
     console.log(`Your move is ${move}`);
+    const computermove = getRandomInt(3)
+    const cmoves = ["r","p","s"]
+    const cmovesrps = cmoves [computermove]
```
Comment (nick-simone) [inline rps.js]: make a better variable name please

**Review by nick-simone — CHANGES_REQUESTED** [review]

File: rps.js (hunk 1 above)
Comment (AndrewMacAskill) [reply rps.js]: Fawk!

File: rps.js (hunk 2 above)
Comment (AndrewMacAskill) [reply rps.js]: FAWk no

**Comment by AndrewMacAskill** [comment]
### ### ### ### ### ### ### ### 
FAWk!
- [ ] #> [****]([url](url))

**Review by nick-simone — APPROVED** [review]

**Pull request merged**